test/ex_ach/field_validator_test.exs

```diff
@@ -0,0 +1,174 @@
+defmodule ExAch.FieldValidatorTest do
+  use ExUnit.Case
+  alias ExAch.FieldValidator
+
+  describe "validating string type" do
+    test "a non alpha numeric string type returns an error" do
+      specifications = [{:field_name, :type, :alpha_num_string}]
+      errors = FieldValidator.validate(123, specifications)
+
```

Comment:
Remove blank lines.  When the test has only 3 lines (or less), they should be grouped.